dlog/kimchi/src/verifier.rs

```diff
@@ -344,7 +349,7 @@ where
             fq_sponge,
             digest,
             oracles,
-            alphas,
```

Comment:
looks like we don't use this `alphas` anywhere (at least not in proof-systems, and not in ocaml-land). Since we need to create the `Alphas` structure within the `oracles()` function we need to return it here. BTW I'd really like to get rid of this `oracles()` function and this `Oracles` struct.